thebrenthaines/cypher-dsl
# Refactor and update of DSL library

* Changed the package structure to be more standard scala, also
* To remove the personalized package name (no offense) in a hopefully fruitful attempt to make this library more mainstream
* Update to support the latest version of cypher
* Update to support scala 2.12
* Add support to `Context` for type names to generate cypher placeholders so query order no longer mangles the query
* Add support for WHERE and WHERE_NOT and most boolean statements
* Completed / corrected support for ORDER_BY
* Other cool updates that make this cool library hopefully a little cooler
